src/sentry/models/integrations/repository_project_path_config.py

```diff
@@ -23,11 +23,21 @@ class Meta:
 
 
 def process_resource_change(instance, **kwargs):
+    from sentry.models import Organization, Project
     from sentry.tasks.codeowners import update_code_owners_schema
 
-    return update_code_owners_schema.apply_async(
-        kwargs={"organization": instance.project.organization, "projects": [instance.project]}
-    )
+    def _spawn_task():
+        try:
+            update_code_owners_schema.apply_async(
+                kwargs={
+                    "organization": instance.project.organization,
+                    "projects": [instance.project],
+                }
+            )
+        except (Project.DoesNotExist, Organization.DoesNotExist):
```

Comment:
I added this because otherwise the signal handler would crash random tests.

this typically happens when the project or organization is being deleted, as that also triggers `post_save`. but even if it's a legitimate bug I think it's not wise to let this error bubble up into random endpoints. this is now much more likely to happen because previously the model deletion triggered this hook before the transaction committed.